### PR TITLE
Add user obc to JH ns

### PR DESCRIPTION
**Pull request by HumairAK**
Resolves: https://github.com/operate-first/support/issues/198#issuecomment-816938594 

I've just added it to the JH namespace following the similar pattern as datacatlog (and pvcs in the JH path).

### CODE REVIEW

**Comment by HumairAK** [comment]
/hold 

would like an OK from @4n4nd and @tumido  since the size request seems a bit large

File: odh/overlays/moc/zero/jupyterhub/obcs/user-yaar-obc.enc.yaml
Comment (4n4nd) [inline odh/overlays/moc/zero/jupyterhub/obcs/user-yaar-obc.enc.yaml:8]: @HumairAK I think we can just use `bucketName` since this is based on a username so we don't have to worry about a duplicate bucket

Comment (HumairAK) [reply odh/overlays/moc/zero/jupyterhub/obcs/user-yaar-obc.enc.yaml:8]: are you proposing in this case we should just have the bucket-name be that of the requester?

Comment (4n4nd) [reply odh/overlays/moc/zero/jupyterhub/obcs/user-yaar-obc.enc.yaml:8]: Yes

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Review by tumido — APPROVED** [review]
/approve

**Comment by harshad16** [comment]
If you want this pr to be merged, please unhold and approve again or add approved label.

**Comment by chauhankaranraj** [comment]
@harshad16 @HumairAK @4n4nd @tumido hey folks could you please take a look at [this](https://github.com/operate-first/support/issues/198#issuecomment-816981568) comment before merging? Just wanted to explicitly make sure this is the right place to store this dataset :)

**Comment by tumido** [comment]
@chauhankaranraj ok, let's discuss that back in the original issue and hold on this PR for now than.

**Comment by HumairAK** [comment]
requester will be going with the alternative/robust option as indicated in the discussion thread: https://github.com/operate-first/support/issues/198#issuecomment-818048689

**Pull request closed without merging**
